src/main/java/com/server/ResponseBodyBuilder.java

```diff
@@ -12,8 +12,12 @@
         this.publicDir = directory;
     }
 
-    public String getBody(RequestParams requestParams) {
-        if (requestParams.getPath().equals("/coffee")) {
+    public String getBody(RequestParams requestParams) throws IOException {
+        if (requestParams.getPath().equals("/file1")) {
```

Comment:
I like the update here! @ranizilpelwar & @shibani, I'm just wondering what your thoughts are on having all of the branches of this conditional look the same, so they follow the same format:

```java
if (requestParams.getPath().equals("<<PATH>>")){
    this.body = <<PATH>>body(requestParams);
}
```

My only thought is that it will help highlight the pattern and aid in future refactorings, but it's totally not necessary, especially if it causes more confusion/redirection.

Reply:
I think the pattern is a good direction as well, and refactor options might become clearer once we look at CRUD.